pokedex/src/components/cuadrilla/styles.module.scss

```
@@ -0,0 +1,337 @@
+  width: 105px;
+  height: 110px;
+  border: 1px solid $fire;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.fire .order {
+  color: $fire;
+}
+
+.fire .namePokemon {
+  background-color: $fire;
+}
+
+.water {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $water;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.water .order {
+  color: $water;
+}
+
+.water .namePokemon {
+  background-color: $water;
+}
+
+.bug {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $bug;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.bug .order {
+  color: $bug;
+}
+
+.bug .namePokemon {
+  background-color: $bug;
+}
+
+.normal {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $normal;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.normal .order {
+  color: $normal;
+}
+
+.normal .namePokemon {
+  background-color: $normal;
+}
+
+.poison {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $poison;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.poison .order {
+  color: $poison;
+}
+
+.poison .namePokemon {
+  background-color: $poison;
+}
+
+.rock {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $rock;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.rock .order {
+  color: $rock;
+}
+
+.rock .namePokemon {
+  background-color: $rock;
+}
+
+.ghost {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $ghost;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.ghost .order {
+  color: $ghost;
+}
+
+.ghost .namePokemon {
+  background-color: $ghost;
+}
+
+.steel {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $steel;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.steel .order {
+  color: $steel;
+}
+
+.steel .namePokemon {
+  background-color: $steel;
+}
+
+.fighting {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $fighting;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.fighting .order {
+  color: $fighting;
+}
+
+.fighting .namePokemon {
+  background-color: $fighting;
+}
+
+.flying {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $flying;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.flying .order {
+  color: $flying;
+}
+
+.flying .namePokemon {
+  background-color: $flying;
+}
+
+.ground {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $ground;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.ground .order {
+  color: $ground;
+}
+
+.ground .namePokemon {
+  background-color: $ground;
+}
+
+.psychic {
+  width: 105px;
+  height: 110px;
+  border: 1px solid $psychic;
+  text-align: center;
+  border-radius: 8px;
+  background-color: $white;
+}
+
+.psychic .order {
+  color: $psychic;
+}
+
+.psychic .namePokemon {
```

Tener cuidado cómo estamos utilizando los nombres en inglés. 

Si ponemos pokemonName, significa "nombre de pokemon"
Si ponemos namePokemon, significa "pokemon de nombre"

```suggestion
.psychic .pokemonName {
```

---

cambiado